tests/particles/particle_04.cc

```diff
@@ -55,7 +55,7 @@ void test ()
     particle.write_data(write_pointer);
 
     const void *read_pointer = static_cast<const void *> (&data.front());
-    Particles::PropertyPool pool(0);
```

Comment:
We still need this, don't we?

Reply:
No, because the new default constructor will create a property pool with 0 properties.

Reply:
Edit: Of course you are right, I did the testing when I had a default of 0, not when I removed the default. Should be fixed now.